<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoShare</class>
 <widget class="QWidget" name="PhotoShare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="toolBarFrame">
     <layout class="QHBoxLayout" name="toolBarFrameHLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QToolButton" name="toolButton_NewAlbum">
        <property name="text">
         <string>Create Album</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/images/add_image24.png</normaloff>:/images/add_image24.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_ViewAlbum">
        <property name="text">
         <string>View Album</string>
        </property>
        <property name="icon">
         <iconset resource="Photo_images.qrc">
          <normaloff>:/images/kview_24.png</normaloff>:/images/kview_24.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_subscribe">
        <property name="text">
         <string>Subscribe To Album</string>
        </property>
        <property name="icon">
         <iconset resource="Photo_images.qrc">
          <normaloff>:/images/album_subscribe.png</normaloff>:/images/album_subscribe.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_SlideShow">
        <property name="text">
         <string>Slide Show</string>
        </property>
        <property name="icon">
         <iconset resource="Photo_images.qrc">
          <normaloff>:/images/kuickshow.png</normaloff>:/images/kuickshow.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="toolBarHSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="sortGroup">
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="sortGroupHLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_YourAlbums">
        <property name="text">
         <string>My Albums</string>
        </property>
        <property name="icon">
         <iconset resource="Photo_images.qrc">
          <normaloff>:/images/my_album_48.png</normaloff>:/images/my_album_48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_SubscribedAlbums">
        <property name="text">
         <string>Subscribed Albums</string>
        </property>
        <property name="icon">
         <iconset resource="Photo_images.qrc">
          <normaloff>:/images/subscribed_album_48.png</normaloff>:/images/subscribed_album_48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_SharedAlbums">
        <property name="text">
         <string>Shared Albums</string>
        </property>
        <property name="icon">
         <iconset resource="Photo_images.qrc">
          <normaloff>:/images/friends_album_48.png</normaloff>:/images/friends_album_48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QScrollArea" name="scrollAreaTop">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContentsTop">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>804</width>
         <height>195</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#scrollAreaWidgetContents{border: none;}</string>
       </property>
       <layout class="QHBoxLayout" name="scrollAreaWidgetContentsTopHLayout">
        <item>
         <spacer name="scrollAreaHSpacerTop">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>139</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QScrollArea" name="scrollAreaBottom">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContentsBottom">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>804</width>
         <height>194</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#scrollAreaWidgetContents{border: none;}</string>
       </property>
       <layout class="QHBoxLayout" name="scrollAreaWidgetContentsBottomHLayout">
        <item>
         <spacer name="scrollAreaHSpacerBottom">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>426</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonHLayout">
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolButton_ViewPhoto">
       <property name="text">
        <string>View Photo</string>
       </property>
       <property name="icon">
        <iconset resource="Photo_images.qrc">
         <normaloff>:/images/kview_24.png</normaloff>:/images/kview_24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
  <include location="Photo_images.qrc"/>
 </resources>
 <connections/>
</ui>
